ledger/src/blockstore.rs

```diff
@@ -6872,18 +6872,27 @@ pub mod tests {
 
         // Write entries
         let gap: u64 = 10;
-        let shreds: Vec<_> = (0..64)
+
+        let keypair = Keypair::new();
+        let reed_solomon_cache = ReedSolomonCache::default();
+        let mut stats = ProcessShredsStats::default();
+        let shreds: Vec<_> = (0u64..64)
             .map(|i| {
-                Shred::new_from_data(
-                    slot,
-                    (i * gap) as u32,
-                    0,
-                    &[],
-                    ShredFlags::empty(),
-                    i as u8,
-                    0,
-                    (i * gap) as u32,
-                )
+                let shredder = Shredder::new(slot, slot - 1, i as u8, 42).unwrap();
```

Comment:
I'm guessing you changed `slot` above from `0` to `1` to avoid wrapping here ? Using `0` as the parent slot for slot `0` is supported / acceptable

Reply:
Yes, this simply looked better than .wrapping_sub()